pkg/volume/empty_dir/empty_dir.go

```diff
@@ -155,20 +155,12 @@ func (plugin *emptyDirPlugin) ConstructVolumeSpec(volName, mountPath string) (*v
 type mountDetector interface {
 	// GetMountMedium determines what type of medium a given path is backed
 	// by and whether that path is a mount point.  For example, if this
-	// returns (mediumMemory, false, nil), the caller knows that the path is
+	// returns (v1.StorageMediumMemory, false, nil), the caller knows that the path is
 	// on a memory FS (tmpfs on Linux) but is not the root mountpoint of
 	// that tmpfs.
-	GetMountMedium(path string) (storageMedium, bool, error)
+	GetMountMedium(path string) (v1.StorageMedium, bool, error)
 }
 
-type storageMedium int
-
-const (
-	mediumUnknown   storageMedium = 0 // assume anything we don't explicitly handle is this
```

Comment:
does this not map reasonably to `v1.StorageMediumDefault`?

Reply:
@liggitt :+1: sounds reasonable to me! I update the PR. PTAL. Thanks!